src/main/kotlin/com/msd/application/dto/GameMapPlanetDto.kt

```diff
@@ -9,6 +9,7 @@ class GameMapPlanetDto(
     val id: UUID,
     @JsonProperty("movement_difficulty")
     val movementDifficulty: Int,
+    @JsonProperty("planet_type")
```

Comment:
Why ist this here? Don't we use camelCase?

Reply:
This is what we receive from the Map Service, and they use snake